shared/js/background/experiments.es6.js

```diff
@@ -0,0 +1,43 @@
+const settings = require('./settings.es6')
+const retentionExperiments = require('../../data/experiments-out')
+
+class Experiment {
+    constructor () {
+        this.variant = ''
+        this.activeExperiment = {}
+    }
+
+    getVariant () {
+            const atbVal = settings.getSetting('atb')
+            if (atbVal && atbVal[atbVal.length - 2].match(/[a-z]/i)) {
+                this.variant = atbVal[atbVal.length - 2] 
+            } else {
+                this.variant = '_'
+            }
+            return this.variant
+    }
+
+    setActiveExperiment () {
+        settings.ready()
+        .then(this.getVariant.bind(this))
+        .then(() => {
+            this.activeExperiment = retentionExperiments[this.variant] || {}
+
+            if (this.activeExperiment.name) {
+                settings.updateSetting('activeExperiment', this.activeExperiment)
+
+                if (this.activeExperiment.settings) {
```

Comment:
We can specify that an experiment should change settings in experiments-out.js. This will come in handy down the road when we want to test different lists.